modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -4629,3 +4637,166 @@ def from_dataframe(cls, df: "ProtocolDataframe") -> "PandasDataframe":
         ErrorMessage.default_to_pandas(message="`from_dataframe`")
         pandas_df = from_dataframe_to_pandas(df)
         return cls.from_pandas(pandas_df)
+
+    def case_when(self, caselist):
+        """
+        Replace values where the conditions are True.
+
+        Parameters
+        ----------
+        caselist : list of tuples
+
+        Returns
+        -------
+        PandasDataframe
+        """
+        cls = type(self)
+        wrapper_put = self._partition_mgr_cls._execution_wrapper.put
+        if (remote_fn := getattr(cls, "_CASE_WHEN_FN", None)) is None:
+
+            def case_when(df, caselist):  # pragma: no cover
+                caselist = [
+                    tuple(
```

Comment:
Why do we have to wrap this in a tuple?

Reply:
According to the doc, caselist is a list of tuples.